.github/workflows/ci.yml

```diff
@@ -639,7 +639,13 @@ jobs:
     container: "${{ matrix.container }}"
 
     steps:
-    - uses: actions/checkout@v3
```

Comment:
This looks like it used to be v3, does v3 work (and v4 not)? Or does it really require v1 like Debian Buster?

Reply:
Oops, thanks for catching this! I overlooked that it worked with `v3` before.